Ship_Game/UI/UIButton_Style.cs

```diff
@@ -52,15 +53,15 @@ public StyleTextures()
 
             public StyleTextures(string normal)
             {
-                Normal  = ResourceManager.Texture(normal);
-                Hover   = ResourceManager.Texture(normal + "_hover");
+                Normal = ResourceManager.Texture(normal);
+                Hover = ResourceManager.Texture(normal + "_hover");
                 Pressed = ResourceManager.Texture(normal + "_pressed");
             }
 
             public StyleTextures(string normal, string hover)
             {
-                Normal  = ResourceManager.Texture(normal);
-                Hover   = ResourceManager.Texture(hover);
+                Normal = ResourceManager.Texture(normal);
+                Hover = ResourceManager.Texture(hover);
```

Comment:
Please restore previous formatting here

Reply:
restored in commit "reverted formatting"